### PR TITLE
fix(sqs): queueUrl property has incorrect region and account for imported queue

### PR DESCRIPTION
**Pull request by ayush987goyal**
fix(sqs): imported queue using incorrect options to generate queueUrl

Currently if we use `fromQueueArn` static helper to import an existing queue, the generated `queueUrl` property uses incorrect account and region. It uses the stack's region/account in which it is imported rather than the one in which it was created.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by ayush987goyal** [comment]
@MrArnoldPalmer Mind taking a look at it? :)

File: packages/@aws-cdk/aws-sqs/test/test.sqs.ts [hunk 1]
```diff
@@ -154,15 +154,15 @@ export = {
...
 
       // WHEN
-      const imports = sqs.Queue.fromQueueArn(stack, 'Imported', 'arn:aws:sqs:us-east-1:123456789012:queue1');
+      const imports = sqs.Queue.fromQueueArn(stack, 'Imported', 'arn:aws:sqs:us-west-2:123456789012:queue1');
 
       // THEN
 
       // "import" returns an IQueue bound to `Fn::ImportValue`s.
-      test.deepEqual(stack.resolve(imports.queueArn), 'arn:aws:sqs:us-east-1:123456789012:queue1');
+      test.deepEqual(stack.resolve(imports.queueArn), 'arn:aws:sqs:us-west-2:123456789012:queue1');
```
Comment (MrArnoldPalmer) [inline packages/@aws-cdk/aws-sqs/test/test.sqs.ts:162]: Can we make a separate test that explicitly passes a region other than the one that is defined for the stack?

**Review by MrArnoldPalmer — DISMISSED** [review]
Some questions for you, if its possible to add these cases as new unit tests that would be awesome!

1. Can `fromQueueArn` be used within an environment agnostic stack? IE when the region of the stack isn't passed.
2. What happens if the user passes the arn as a token? Does parsing the arn fail?

These may not differ from current behavior but since this module is marked as stable I want to make sure we aren't gonna break existing apps. This change overall makes sense though.

File: packages/@aws-cdk/aws-sqs/test/test.sqs.ts (hunk 1 above)
Comment (ayush987goyal) [reply packages/@aws-cdk/aws-sqs/test/test.sqs.ts]: Yeah it makes sense to have specific test-case. Added.

**Comment by ayush987goyal** [comment]
> 1. Can fromQueueArn be used within an environment agnostic stack? IE when the region of the stack 
 isn't passed.
    2. What happens if the user passes the arn as a token? Does parsing the arn fail?

1. Yes, so in our test-cases we create region/account agnostic stacks only (since we do not explicitly pass account and region). [Ref](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/core/lib/environment.ts)
2. Yes, `parseArn` methods returns correct data for tokens too. [Ref](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/core/lib/arn.ts#L140)

**Review by nija-at — APPROVED** [review]

**Pull request merged**